CHANGELOG.md

```diff
@@ -10,6 +10,9 @@ Every entry has a category for which we use the following visual abbreviations:
 
 ## [Unreleased]
 
+- 🐞 Building VAST now properly respects the environment variables `LDFLAGS`
+  and `ARFLAGS`, instead of passing the `LDFLAGS` to both `ld` and `ar`.
+
```

Comment:
Thanks! Just a quick first-time comment. We try to have the following format for bugfixes:

1. First describe the bug as concise as possible.
2. Then describe the fix.